src/pages/connections/farmlands.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: ../../layouts/ConnectionLayout.astro
+description: Farmlands has partnered with Centrapay to deliver new ways for your business to authorise and process Farmlands Card payments that are faster, easier, and more secure than ever.
+---
+import Grid from '../../components/Grid.astro';
+import Card from '../../components/Card.vue';
+
+In partnership with Centrapay, Farmlands has two new digital solutions that enable Card Partners to authorise Card payments in real-time, and process invoices to Farmlands faster. There are no transactional costs for Card Partners using these solutions.
+
+Your business can either fully integrate the Farmlands solution to your retail point of sale system, or access a secure web portal when accepting Farmlands Card as a form of payment for a sale anytime, anywhere.
+
+![Farmlands card flow](/farmlands-flow.png)
+
+## Solutions
+<Grid>
+  <Card
+    title='1. POS API'
```

Comment:
Why is everything passed in as a prop?

A card component is a well-defined component in UI. It's not opinionated on how the content is laid out or what can go in it.

It's more there to add padding, edging, shadowing etc.
Is there a way you can use a slot, or should this be a more specific component

Reply:
I'm reusing an existing component. I'm not against updating it but I would like to do it in a follow up PR